nixos/modules/services/networking/spruned.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.spruned;
+
+  cliArgs = "--datadir ${cfg.dataDir} --network bitcoin.${cfg.network} ${cfg.extraArguments}";
+in
+{
+  options = {
+    services.spruned = {
+      enable = mkEnableOption "The spruned lightweight Bitcoin pseudonode daemon service";
```

Comment:
You might be sad that you used a capital "T" here.

` description = "Whether to enable ${name}.";` https://github.com/NixOS/nixpkgs/blob/master/lib/options.nix#L68

Reply:
😅